README.md

```diff
@@ -60,12 +60,13 @@ jobs:
         os:
           - macos-latest
           - ubuntu-latest
-          - windows-latest
         ocaml-version:
-          - 4.11.0
-          - 4.10.1
-          - 4.09.1
-          - 4.08.1
+          - ocaml-base-compiler.4.12.0
```

Comment:
Wouldn't the previous version numbers work as well?

Reply:
For now, no, but we can support it. (I didn't have much incentive to support it, so I did not implement that out to keep the code simple.)

Reply:
I think keeping the current format is beneficial for users: it's short, which helps users to tests the packages on multiple versions; and it's the same as in v1, which helps currents users to migrate.

Reply:
It's understandable that it makes the migration easier. However, I want to be as close as possible to the new package naming style introduced in 4.12.

Reply:
@psafont I just re-added semver style version matching support! (For base compilers, not variants, you can just use `4.12.x`. This greatly reduces maintenance costs.)

Reply:
@Gbury Perhaps you are also interested in this change.

Reply:
good to know, ^^